Sprint-2/implement/bmi.js

```diff
@@ -13,3 +13,22 @@
 // Given someone's weight in kg and height in metres
 // Then when we call this function with the weight and height
 // It should return their Body Mass Index to 1 decimal place
+
+function extract1decimal(x){
+    return Math.round(x*10)/10
```

Comment:
Nice! There's also a toFixed function in javascript which can help you accomplish this without having to create your own.